lib/usb_connection.js

```diff
@@ -175,12 +170,18 @@ USB.Connection.prototype._close = function(callback) {
   }
 
   self.closed = true;
-  self.intf.release(true, function() {
-    self.device.close();
-    if (typeof callback === 'function') {
-      callback();
-    }
-  }.bind(self));
+  self.epIn.stopPoll(function() {
```

Comment:
Adding `stopPoll` is apparently the sanitary thing to do when closing the connection.